change/@fluentui-react-tree-a886fa7c-b0f9-4646-8bd7-e9be02201229.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33277&commitId=3feccecc006f10111ce289a41c505d187b87ec26&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.size+image+badge.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4933| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2829| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>